### ISSUE TITLE
Datasets

### ISSUE THREAD

**Issue by cmdllx** [google-research/long-range-arena#12]

**Comment by cmdllx** [comment]
I have downloaded lra_release.gz and unzipped this file, but I do not see the the datasets of the six tasks.

**Comment by vanzytay** [comment]
Hey, IMDb and Cifar are available in tensorflow datasets. Hence, we do not provide the raw files. The rest of the datasets should be in the released zip file.

**Comment by cmdllx** [comment]
I get a lra_release file named 'lra_release' after unzipping the released zip file.
but I do not find the path metioned in the script:--output_dir=$HOME/lra_data/listops/

**Comment by ni0356** [comment]
I second this issue and the response from @vanzytay does not seem to address the issue. 

I can download the lra_release.gz which is of size 8.23 GB and when it is unzipped the files is then 18.6 GB but the file structure required to run the code is lost and the file is not able to be opened. 

I have also tried to download the files directly from the cloud but receive an error 'AccessDeniedException: 403 <user> does not have storage.objects.list access to the Google Cloud Storage bucket.'

**Comment by vanzytay** [comment]
Hi,

There are two options here.

1) Create file directories to match the output_dir args. Then copy the files from unzipped files into the created dirs.
2) Simply modify the output args to match the file directories.

Tldr; The path from the zipped dataset release **does not** match the output dir args. you have to match them yourself but this should be quite straightforward.

**Comment by ni0356** [comment]
There are no 'files' when the referenced zip is unzipped. It is just a single file and it cannot be loaded. I would be happy to recreate the directory structure but that isn't the issue. The issue is that the unzipping results in a single file and not multiple files.

**Comment by vanzytay** [comment]
@ni0356 it's a gz file and if you untar the gz file you'll find the directories.

**Comment by jamesal1** [comment]
I'd suggest changing the name to .tar.gz then so that people won't be confused.

**Comment by cmdllx** [comment]
[自动回复]你好，我已收到你的邮件。
